client/src/index.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './components/App';
+import App from './App';
```

Comment:
Shouldn't import location be './App/App'?

Reply:
There is an index file inside each folder which points to the main javascript file

Reply:
Oh, alright. Thanks!